public/index.js

```diff
@@ -0,0 +1,7 @@
+document.addEventListener('DOMContentLoaded', (event) => {
```

Comment:
Added an event listener to trigger when page is loaded to fetch Open Ai response.